pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+Include a summary of the change and what was done. Also include relevant motivation and context for the change. If any, list dependencies are required for this change.
+
+Include the Asana link, or relevant issue. (https://app.asana.com/0/187002107997407/list)
+
+Don't forget to add reviewers on github and mention the PR in the appropriate discord channel.
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions and details so the changes can be tested if required.
+
+# Documentation
+
+Include links to changes in documentation on confluence that you have created, if it was required.
+
+https://uwarg-docs.atlassian.net/wiki/spaces/ZP/overview?homepageId=20217925
+
+# Checklist:
+
+- [ ] The changes have been well commented, particularly in hard-to-understand areas.
+- [ ] The code has been tested on hardware, either by me or somone else.
+- [ ] corresponding changes to documentation have been created links to this documentation are provided within the pull request.
+- [ ] The changes generate no new warnings and compile.
```

Comment:
Can we include screenshots in PR's of the code compiling and running?
++ make sure to mention who tested on hardware?

Reply:
That's a good idea, they can tag the person that tested it, and if they used console or something to view/validate they can also drop ss of that